### ISSUE TITLE
Improve exporter documentation references in frontend developer docs

### ISSUE THREAD

**Issue by christian-konrad** [camunda/camunda-docs#3620]
To answer the many questions of users regarding exporters and their config (to achieve near real-time results), we should either extend the section onhttps://docs.camunda.io/docs/next/apis-tools/frontend-development/task-applications/user-task-lifecycle/#listen-to-life-cycle-events or add a dedicated page to the frontend developer docs.

**Comment by npepinpe** [comment]
I don't think this is really relevant anymore, what with the user task listeners largely replacing this. We could still document it for older versions of course, but it's a question of how important this is, and how much effort we want to put.

I'll leave it up to the HTO team to decide, but my recommendation would be to close it until someone requests it.
